integration-test/ui/feedback.js

```diff
@@ -0,0 +1,97 @@
+const harness = require('../helpers/harness')
+const EXAMPLE_URL = `https://example.com/test`
+
+let browser
+let bgPage
+let feedbackUrl
+let feedbackPage
+
+describe('feedback flow', () => {
+    beforeAll(async () => {
+        ({ browser, bgPage } = await harness.setup())
+
+        const extId = await bgPage.evaluate(() => chrome.runtime.id)
+
+        feedbackUrl = `chrome-extension://${extId}/html/feedback.html`
+    })
+    afterAll(async () => {
+        await harness.teardown(browser)
+    })
+    beforeEach(async () => {
+        feedbackPage = await browser.newPage()
+
+        await feedbackPage.setRequestInterception(true)
+        feedbackPage.on('request', async (req) => {
+            if (req.url().match(/duckduckgo\.com\/feedback\.js/)) {
+                await req.respond({
+                    status: 200,
+                    contentType: 'text/json',
+                    body: '{ "status": "success" }'
+                })
+            } else {
+                await req.continue()
+            }
+        })
+    })
+    afterEach(async () => {
+        await feedbackPage.close()
+    })
+    describe('broken site', () => {
+        it('should prefill the site if we pass it via the query string', async () => {
+            await feedbackPage.goto(`${feedbackUrl}?broken=1&url=${encodeURIComponent(EXAMPLE_URL)}`)
+
+            const val = await feedbackPage.evaluate(() => document.querySelector('.js-feedback-url').value)
+
+            expect(val).toMatch(EXAMPLE_URL)
+        })
+        it('should be possible to submit after everything has been filled out', async () => {
+            await feedbackPage.goto(`${feedbackUrl}?broken=1`)
+            await feedbackPage.waitForSelector('.js-feedback-submit')
+
+            // clicking times out for some reason, so we try and enter instead
+            // (stolen from https://github.com/GoogleChrome/puppeteer/issues/1805#issuecomment-418965009)
+            await feedbackPage.focus('.js-feedback-submit')
+            await feedbackPage.keyboard.type('\n')
```

Comment:
Not sure why this was needed. Seems to work.

It might get fixed if we change to a newer version of puppeteer, but I didn't want to get into a rabbit hole.